src/lib.rs

```diff
@@ -258,65 +279,100 @@ impl Watch {
             };
         }
     }
+
+    fn is_excluded_path(&self, path: &Path) -> bool {
```

Comment:
You can clearly see the non sense repeated code here. It shows the 2 vectors are identical